packages/Box/__tests__/Box.spec.jsx

```diff
@@ -17,10 +15,6 @@ describe('Box', () => {
     return box.find(box.props().tag)
   }
 
-  beforeEach(() => {
-    mockMatchMedia(575)
-  })
-
   it('renders', () => {
     const box = doMount()
```

Comment:
The only reason we were doing `mount` in the Box tests was because of the presence of the `Responsive` component, so that we could render its children in the tests. Now that there is no more `Responsive` component in Box, I believe all of these tests can be changed to use `shallow`.

`shallow` rendering is much preferred when we can do it because it is faster than `mount`.

So, these tests should be modified to use `shallow` instead of `mount`.